backend/db/migrate/20250814120058_add_text_to_documents.rb

```
@@ -0,0 +1,5 @@
+class AddTextToDocuments < ActiveRecord::Migration[8.0]
```

RuboCop (1.76.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)